bootstrap/kubeadm/internal/ignition/clc/clc_test.go

```diff
@@ -73,6 +72,11 @@ func TestRender(t *testing.T) {
 		{
 			desc: "renders valid Ignition JSON",
 			input: &cloudinit.BaseUserData{
+				BootCommands: []bootstrapv1.BootCommand{
+					{
+						"boot-command", "another-boot-command",
+					},
+				},
```

Comment:
This does not make sense to get set right? Because ignition does not have `bootCommands` and we block this getting set?

Reply:
You're right, it doesn't make sense to have that there.